src/test/java/caculator/domain/Calculator.java

```diff
@@ -1,13 +1,22 @@
 package caculator.domain;
 
+import java.util.Objects;
+
 public class Calculator {
 
     private Calculator() {
         throw new AssertionError();
     }
 
     public static int add(String stringNumbers) {
-        Numbers numbers = StringSplitter.split(stringNumbers);
-        return numbers.sum();
+        if (cannotCalculate(stringNumbers)) {
+            return 0;
+        }
+
+        return Numbers.from(stringNumbers).sum();
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        StringNet stringNet = StringNetFactory.getStringNet(stringNumbers);
        return Numbers.from(stringNet.strain(stringNumbers)).sum();
```

Reply:
StringNetFactory에 대한 의존을 한 단계(?) 위로 올리니 결합도가 낮아졌어요!
클린코드 강의 때 테스트하기 좋은 코드로 변경하는 방법에서 배웠던 내용인데 연습이 부족했네요 😂